.github/workflows/ci.yml

```
@@ -51,12 +51,12 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.11'
+          python-version: '3.14'
 
       - name: Setup Node
         uses: actions/setup-node@v3
```

actionlint (1.7.9)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)